launcher/clabernetes.go

```diff
@@ -55,11 +69,16 @@ func StartClabernetes() {
 var clabernetesInstance *clabernetes //nolint:gochecknoglobals
 
 type clabernetes struct {
-	ctx context.Context
+	ctx    context.Context
+	cancel context.CancelFunc
 
 	appName string
 
-	logger claberneteslogging.Instance
+	logger             claberneteslogging.Instance
+	containerlabLogger claberneteslogging.Instance
+	nodeLogger         claberneteslogging.Instance
+
+	containerIDs []string
```

Comment:
I wonder, when are we having >1 container in a launcher pod?

Reply:
literally no idea :P but figured if/when that happens we'd want to care about them all. obviously we can easily just watch a single id at some point too!